helpers.py

```diff
@@ -102,3 +102,152 @@ def downloadCreateDataList(itemList,site,format,dataPath):
 
     except Exception:
         raise Exception(traceback.format_exc())
+
+#============================
+#Amino Percentage based
+# a Sequence and a Letter
+#============================
+def calcAminoPercentage(seq,letter):
+  counter=0
+  for alphabet in seq:
+    if (alphabet==letter):
+      counter=counter+1
+  
+  return ((counter/len(seq))*100)
+
+#============================
+#Rounds Floating Points Number
+#============================
+def roundFloatingpoint(num):
+  return (np.round(num*1000)/1000)
+
+#============================
+#Removes Undesired Character
+#============================
+def removeSpaceNewLine(seq):
+  return ((seq.replace('\n','')).replace(' ',''))
+
+#============================
+# Creating Directory (if does not exist)
+# Clearing the file (if already exist) 
+#============================
+def clearFile(fileAddress):
+    #Trying to create directories (If does not exist)
+    try:
+        open((os.getcwd()+fileAddress), 'w').close()
+    except:
+        newPath=(fileAddress).split("/")
+        path = ("/".join(newPath[:(len(newPath)-1)]))
+        os.mkdir(os.getcwd()+path)
+    finally:
+        open((os.getcwd()+fileAddress), 'w').close()
+
+#============================
+#Calculates AAC(each Sequence)
+#============================
+def calcSeqAAC(sequence,letters):
+    sequence = removeSpaceNewLine(sequence)
+    aminoPercentage=[]
+
+    for amino in letters:
+
+        aminoPercentage.append(
+            roundFloatingpoint(
+                calcAminoPercentage(sequence,amino)))
+
+    return aminoPercentage
+
+#============================
+#Calculates AAC(each File)
+#============================
+def calcAAC(aminoLetters,source,label,destination):
+    try:
+        # Reading Source Transporter
+        sourceHandler=open(source,'r')
+        sourceData=(sourceHandler.read()).split("\n")
+        sourceHandler.close()
+
+        #Calculating, Writing to File
+        for line in sourceData:
+            if '>' not in line:
+                #Calculating the AAC feature and Writes it to aac.csv
+                destination.write('\n'+
```

Comment:
non portable (check all occurrences of `+`, `/` and `\n` in string manipulations)